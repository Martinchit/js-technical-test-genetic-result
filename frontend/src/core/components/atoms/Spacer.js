import styled from 'styled-components';

export const Spacer = styled.hr`
  height: 1px;
  width: 100%;
  background-color: ${({ theme }) => theme.colors.grey};
`;
