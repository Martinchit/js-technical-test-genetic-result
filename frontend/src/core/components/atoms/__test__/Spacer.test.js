import React from "react";
import { shallow } from 'enzyme';
import 'jest-styled-components';

import { Spacer } from "../Spacer";
import { Theme } from "../../../lib/theme/globalStyle";

describe("Spacer", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<Spacer theme={Theme}  />)
    expect(wrapper).toMatchSnapshot()
  })
})