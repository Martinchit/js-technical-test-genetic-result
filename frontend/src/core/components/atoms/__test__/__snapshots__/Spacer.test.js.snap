// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spacer renders correctly 1`] = `ShallowWrapper {}`;

exports[`Text renders correctly 1`] = `ShallowWrapper {}`;
